README.md

```diff
@@ -27,6 +23,14 @@ The Prometheus operator includes, but is not limited to, the following features:
 
 For an introduction to the Prometheus Operator, see the [getting started](https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/user-guides/getting-started.md) guide.
 
+## Project Status
+
+The operator in itself is considered to be production ready. Please refer to the Custom Resource Defintion (CRD) versions for the status of each CRD:
```

Comment:
```suggestion
The operator in itself is considered to be production ready. Please refer to the Custom Resource Definition (CRD) versions for the status of each CRD:
```

Reply:
thanks for the thorough review :) Will fix it upstream.